test/lib/BedrockTester.cpp

```diff
@@ -158,6 +149,35 @@ string BedrockTester::startServer(bool wait) {
                  << _controlPort << ") to start server, things will probably fail." << endl;
         }
 
+#ifdef VALGRIND
+    #define xstr(a) str(a)
+    #define str(a) #a
+
+    list<string> valgrind = SParseList(xstr(VALGRIND), ' ');
```

Comment:
I've read this https://gcc.gnu.org/onlinedocs/gcc-4.8.5/cpp/Stringification.html but am not fully connecting the dots just yet, will file away as :magic: for now.

Reply:
It's magic.